src/components/layout/Layout.js

```diff
@@ -38,9 +38,7 @@ const NavbarRow = styled(Navbar)`
   }
 `;
 
-const FooterKoro = styled(KoroSection)`
-  margin-top: 4em;
-`;
+const FooterKoro = styled(KoroSection)``;
```

Comment:
Koro is broken:

<img width="1347" alt="screenshot 2018-12-13 at 8 00 41" src="https://user-images.githubusercontent.com/1412709/49918868-47129e80-fead-11e8-8a71-802f66370109.png">